gkr_iop/src/precompiles/lookup_keccakf.rs

```diff
@@ -129,6 +129,9 @@ pub struct KeccakWitCols<T> {
     pub nonlinear: [T; 200],
     pub chi_output: [T; 8],
     pub iota_output: [T; 200],
+    pub input_ram_start_addr: [T; 1],
+    pub cur_ts: [T; 1],
+    pub input_ram_read_ts: [T; KECCAK_INPUT32_SIZE],
```

Comment:
2 extra field `cur_ts`, `input_ram_read_ts` is for mem consistency check for offline-memory-check protocol